File: Python praktikum/prak4/Soal1.py
bebek = 0
k = 8

for i in range(k):
    bebek = bebek + int(input("Masukan Berat Bebek ke-"+ str(i+1)+ " "))
                 

rtotal = float(input("Masukan rata rata total 10 Bebek = "))
sbh = int(input("Masukan selisih 2 bebek hilang = "))

jbh = 10 * rtotal - bebek
b9 = (sbh + jbh)/2
b10 = jbh - b9

print("Berat Bebek Ke - 9 = ",b9)
print("Berat Bebek Ke - 10 = ",b10)

File: java praktikum/prak11/JavaApplication19/src/javaapplication19/Soal1.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package javaapplication19;

import java.util.Scanner;

/**
 *
 * @author Nrhys
 */
public class Soal1 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        double tbb, bebek, rtotal, sbh, jbh, b9, b10;
        Scanner input = new Scanner(System.in);
        tbb = 0;
        
        for(int i = 0; i < 8; i++){
            System.out.println("Masukan Berat Bebek ke-" + (i+1));
            bebek = input.nextDouble();
            tbb += bebek;
            
        }
        
        System.out.println("Masukan Rata Rata Total 10 bebek = ");
        rtotal = input.nextDouble();
        
        System.out.println("Masukan Selisih 2 Bebek yang Hilang = ");
        sbh = input.nextDouble();
        
        jbh = 10 * rtotal - tbb;
        b9 = (sbh + jbh)/2;
        b10 = jbh - b9;
        
        System.out.println("Berat Bebek Ke - 9 = "+b9);
        System.out.println("Berat Bebek Ke - 10 = "+b10);
    }
    
}
